ov/server_hardware_type.go

```diff
@@ -70,6 +70,41 @@ type BiosSetting struct {
 	WarningText     string       `json:"warningText,omitempty"`     // "warningText":""
 }
 
+type serverDisplaySchema struct {
+	Args        []args `json:"args,omitempty"`        // "args":{},
+	HelpText    string `json:"helpText,omitempty"`    // "helpText":"",
+	Label       string `json:"label,omitempty"`       // "label":"",
+	SettingType string `json:"settingType,omitempty"` // "settingType":"",
+}
+
+type args struct {
```

Comment:
Same comment and below where you have used smallcase for struct names

Reply:
done